### PR TITLE
admin docs: upgrade pure docker to 3.21

### PR DESCRIPTION
**Pull request by uwedeportivo**
describes upgrade, what's new and links to diff commit

### CODE REVIEW

File: doc/admin/updates/pure_docker.md [hunk 1]
```diff
@@ -9,6 +9,18 @@ Upgrades should happen across consecutive minor versions of Sourcegraph. For exa
...
+
+To upgrade, please perform the changes in the following diff:
+
+https://github.com/sourcegraph/deploy-sourcegraph-docker/commit/617eb56f1c102f20b6835d176df6797e6f5db53d
+
+### Note new services
+
+This upgrade includes a new code-intel DB (`deploy-codeintel-db.sh`) and a new service `minio` (`deploy-minio.sh`) 
+to store LSIF indices.
+There is a new environment variable for frontend and frontend-internal called `CODEINTEL_PGHOST`.
```
Comment (slimsag) [inline doc/admin/updates/pure_docker.md:22]:
```suggestion
There is a new environment variable for frontend and frontend-internal called `CODEINTEL_PGHOST`.

(both of these changes are described exactly in the diff above)
```

just adding clarifying point

**Review by slimsag — APPROVED** [review]
you rock - I let one suggestion inline, and a comment over here: https://github.com/sourcegraph/deploy-sourcegraph-docker/commit/617eb56f1c102f20b6835d176df6797e6f5db53d#r43884277

I think we will need to fix that (and force push the commit / update the diff link here), otherwise LGTM

**Comment by uwedeportivo** [comment]
i did what dave told me :-)

**Comment by uwedeportivo** [comment]
amended the diff commit and updated the commit hash here

**Pull request merged**